src/states/game/game.gd

```diff
@@ -35,17 +33,15 @@ const COPY_ROUND_DOWNTIME = 3
 @onready var card_viewer: Control = %CardViewer
 @onready var hand_bounds: Control = %HandBounds
 
-@onready var lane_drops: Array[Area2D] = [
-	$AttackDropPoints/Lane0, $AttackDropPoints/Lane1, $AttackDropPoints/Lane2,
-	$DefenseDropPoints/Lane3, $DefenseDropPoints/Lane4, $DefenseDropPoints/Lane5,
-	$InfoDropPoint/LaneInfo,
-]
+@onready var drop_points: Node2D = %DropPoints
+@onready var blue_castle_door: Vector2 = $DefenseBridgePoint.position
+@onready var red_castle_door: Vector2 = $AttackBridgePoint.position
 
 @onready var win_sound: AudioStreamPlayer = $WinSound
 @onready var draw_sound: AudioStreamPlayer = $DrawSound
 
 
-
+var drop_world_pos: Dictionary
```

Comment:
Not a fan of the name. Perhaps `grid_to_world_pos`?

Reply:
GDScript doesn't support strict typing of dictionaries but I'd put in a comment detailing the actual typing:

```gdscript
var grid_to_world_pos: Dictionary # Dictionary[Vector2i, Vector2]
```